docs/community/release-process.rst

```diff
@@ -0,0 +1,51 @@
+Release Process and Rules
+=========================
+
+.. versionadded:: v2.6.2
+
+Starting with the version to be released after ``v2.6.2``, the following rules 
+will govern and describe how the Requests core team produces a new release.
+
+Major Releases
+--------------
+
+A major release will include breaking changes. When it is versioned, it will 
+be versioned as ``vX.0.0``. For example, if the previous release was 
+``v10.2.9`` the next version will be ``v11.0.0``.
+
+Breaking changes are changes that break backwards compatibility with prior 
+versions. If the project were to change the ``text`` attribute on a 
+``Response`` object to a method, that would only happen in a Major release.
+
+Major releases may also include miscellaneous bug fixes and upgrades to 
+vendored packages.
+
+Minor Releases
+--------------
+
+A minor release will not include breaking changes but may include 
+miscellaneous bug fixes and upgrades to vendored packages. If the previous 
+version of Requests released was ``v10.2.9`` a minor release would be 
```

Comment:
This example is potentially somewhat ambiguous, because Python used to operate under the rule that after vX.Y.9 came vX.Y+1.0, which is what this *could* look like.